src/game_api/sound_manager.hpp

```diff
@@ -1,18 +1,24 @@
 #pragma once
 
 #include <array>
+#include <cstddef>
+#include <cstdint>
 #include <functional>
+#include <list>
+#include <new>
 #include <optional>
 #include <string>
 #include <string_view>
+#include <type_traits>
 #include <unordered_map>
+#include <utility>
 #include <variant>
 #include <vector>
 
+#include "D:/zollk/Documents/Projects/llvm-project/build/Release/lib/clang/13.0.1/include/stddef.h"
```

Comment:
don't think this will work correctly (?)

Reply:
Yeah, idk why those came in there. How did you find them so quick  xD

Reply:
well, most of the changes are adding/removing includes, so any problems would be catch by the compiler anyway, so while scrolling thru the changes, my eyes just catch that super long include line :smile: